README.md

```diff
@@ -1,2 +1,13 @@
-# drag-and-drop-external-tasks-to-react-gantt-chart
-A quick-start React project that allows you to drag and drop tasks from an external Tree view component to the React Gantt Chart of Syncfusion.
+# Drag and Drop Items from External Source to React Gantt Chart
+
+A quick-start project that allows you to drag and drop an item from an external TreeView component to the React Gantt Chart row. In this project, the dragged item is updated as a resource for the Gantt record using `nodeDragStop` event of TreeView control. 
```

Comment:
Is this content reviewed?

Reply:
Hi @UmaMaheswariChandrabose ,

Not reviewed

Regards,
Vinoth Kumar S